src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -623,4 +623,52 @@ public void executeQuery_incorrectQuery_doesNothing() {
 
     Assert.assertTrue(notificationId == 0);
   }
+
+  /** Tests function buildAnswerFull(). */
+  @Test
+  public void buildAnswer_validResultSet_successfulBuild() {
```

Comment:
This test looks good - think through other test cases you might support. For instance, when SQL columns are missing, or the comments are empty, or the answer is empty, etc. You'll want to be able to look at your test names and have confidence that all the corner cases of the functionality is accounted for.

Reply:
Will think through them!